main.go

```diff
@@ -29,60 +22,81 @@ func dbConn() (db *sql.DB) {
 }
 
 func Index(w http.ResponseWriter, r *http.Request) {
+
+	n := Funcionario{}
+	SlcFuncionario := []Funcionario{}
```

Comment:
variables should not start with uppercase, also, what "Slc" means?

Reply:
I changed it to sliceEmployee